examples/closed_coil/Main.cpp

```diff
@@ -100,7 +100,7 @@ main(int argc, char * argv[])
   solver_options.SetParam("Tolerance", float(1.0e-13));
   solver_options.SetParam("AbsTolerance", float(1.0e-16));
   solver_options.SetParam("MaxIter", (unsigned int)500);
-  solver_options.SetParam("PrintLevel", 2);
+  solver_options.SetParam("PrintLevel", 0);
```

Comment:
Might be worth adding a utility to get a (default) PrintLevel based on the current log level, to avoid users having to specify verbosity in multiple ways?